code/game/objects/items/storage/toolboxes/_toolbox.dm

```diff
@@ -0,0 +1,179 @@
+	current_interactions += 1
+	picked_tool.melee_attack_chain(user, interacting_with, list2params(modifiers))
+	current_interactions -= 1
+
+	if (QDELETED(picked_tool) || picked_tool.loc != user || !user.CanReach(picked_tool))
+		current_interactions = 0
+		return
+
+	if (current_interactions)
+		return
+
+	if (istype(picked_tool, /obj/item/weldingtool))
+		var/obj/item/weldingtool/welder = picked_tool
+		if (welder.welding)
+			welder.attack_self(user)
+
+	atom_storage.attempt_insert(picked_tool, user)
+
+/obj/item/storage/toolbox/update_overlays()
+	. = ..()
+	if(has_latches)
+		. += latches
+
+/obj/item/storage/toolbox/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] robusts [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!"))
+	return BRUTELOSS
+
+//repairbot assembly
+/obj/item/storage/toolbox/tool_act(mob/living/user, obj/item/tool, list/modifiers)
+	if(!istype(tool, /obj/item/assembly/prox_sensor))
+		return ..()
+	var/static/list/allowed_toolbox = list(
+		/obj/item/storage/toolbox/artistic,
+		/obj/item/storage/toolbox/crafter,
+		/obj/item/storage/toolbox/electrical,
+		/obj/item/storage/toolbox/emergency,
+		/obj/item/storage/toolbox/mechanical,
+		/obj/item/storage/toolbox/syndicate,
+	)
+
+	if(!is_type_in_list(src, allowed_toolbox) && (type != /obj/item/storage/toolbox))
+		return ITEM_INTERACT_BLOCKING
+	if(contents.len >= 1)
+		balloon_alert(user, "not empty!")
+		return ITEM_INTERACT_BLOCKING
+	var/static/list/toolbox_colors = list(
+		/obj/item/storage/toolbox = "#445eb3",
+		/obj/item/storage/toolbox/emergency = "#445eb3",
+		/obj/item/storage/toolbox/electrical = "#b77931",
+		/obj/item/storage/toolbox/artistic = "#378752",
+		/obj/item/storage/toolbox/crafter = "#9D3282",
+		/obj/item/storage/toolbox/syndicate = "#3d3d3d",
+	)
+	var/obj/item/bot_assembly/repairbot/repair = new
+	repair.toolbox = type
+	var/new_color = toolbox_colors[type] || "#445eb3"
+	repair.set_color(new_color)
+	user.put_in_hands(repair)
+	repair.update_appearance()
+	repair.balloon_alert(user, "sensor added!")
+	qdel(tool)
+	qdel(src)
+	return ITEM_INTERACT_SUCCESS
+
+
+/obj/item/clothing/head/costume/snakeeater
+	name = "strange bandana"
+	desc = "A bandana. It seems to have a little carp embroidered on the inside, as well as the kanji '魚'."
+	icon_state = "snake_eater"
+	inhand_icon_state = null
+	clothing_traits = list(TRAIT_FISH_EATER)
+
+/obj/item/clothing/head/costume/knight
+	name = "fake medieval helmet"
+	desc = "A classic metal helmet. Though, this one seems to be very obviously fake..."
+	icon = 'icons/obj/clothing/head/helmet.dmi'
+	worn_icon = 'icons/mob/clothing/head/helmet.dmi'
+	icon_state = "knight_green"
+	inhand_icon_state = "knight_helmet"
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
+	dog_fashion = null
```

Comment:
...I see these were in the `toolbox.dm` file before, after the `/obj/item/storage/toolbox/guncase/cqc` in which they spawn.
Wherever they're going, the new base toolbox file probably isn't a good place for them.

Try https://github.com/tgstation/tgstation/blob/master/code/modules/clothing/head/hat.dm maybe?

Reply:
`/clothing/head/costume.dm` is way more fitting